src/components/Events/ConferenceCard.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react'
+import styled from 'styled-components'
+import breakpoint from 'styled-components-breakpoint'
+import remcalc from 'remcalc'
+import Image from '../Common/Image'
+
+import Anchor from '../Common/Anchor'
+
+import { Link } from 'gatsby'
+
+import { Row, Col } from '../grid'
+import StyledLink from '../Common/StyledLink'
+import { CardTitle, BodyPrimary } from '../Typography'
+
+const BlurbWrapper = styled.div`
+  ${breakpoint('tablet')`
+    padding-top: ${remcalc(24)};
+  `};
+`
+
+const StatusWrapper = styled.div`
+  padding-top: ${remcalc(12)};
+  padding-bottom: ${remcalc(12)};
```

Comment:
All of these values are in `theme`, please use them here instead of hard coding values: 

https://github.com/yldio/yld.io/blob/ef7545ede2841818af0757282deb247202b6fab4/src/utils/theme.js#L27

Reply:
Also here: https://github.com/yldio/yld.io/pull/353/files#diff-c12cbaaa077445ec945f2cece1dd3555R17

Reply:
I'll need to do the same to EventCard too.